src/Symfony/Bundle/FrameworkBundle/Secrets/SecretsBroker.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Secrets;
+
+class SecretsBroker extends BaseSecretsHandler
```

Comment:
Make everything final if you can. If you can't because of a unit test mock, use `@final` in phpdoc above it.

Reply:
👍 will do